sequencing-server/src/routes/command-expansion.ts

```diff
@@ -430,18 +804,30 @@ commandExpansionRouter.post('/expand-all-activity-instances', async (req, res, n
     const seqIdToSimActivityId: Record<string, Set<number>> = {};
 
     for (const row of seqToSimulatedActivity.rows) {
+      logger.info(`POST /command-expansion/expand-all-activity-instances:\n` + JSON.stringify(row));
+
       if (seqIdToSimActivityId[row.seq_id] === undefined) {
         seqIdToSimActivityId[row.seq_id] = new Set();
       }
 
       seqIdToSimActivityId[row.seq_id]!.add(row.simulated_activity_id);
+      logger.info(
+        `POST /command-expansion/expand-all-activity-instances:\n` +
+        row.seq_id +
+        ' -> ' +
+        seqIdToSimActivityId[row.seq_id]?.size,
+      );
     }
 
+    logger.info(`POST /command-expansion/expand-all-activity-instances:\n` + JSON.stringify(seqIdToSimActivityId));
+
     // If the user has created a sequence, we can try to save the expanded sequences when an expansion runs.
+    logger.info('ORIGINAL SIMULATED ACTIVITIES: ' + JSON.stringify(simulatedActivities));
     for (const seqRow of seqRows.rows) {
       const seqId = seqRow.seq_id;
       const seqMetadata = seqRow.metadata;
 
+      // move this outside of the loop? why not use simulated activities from before? they use similar loaders (the gql query is the same, and we override an existing variable...why not delete?)
```

Comment:
Has this question been addressed?

Reply:
Ensure this is in #1647. Otherwise, it can be resolved.